Competition2018/src/main/java/competition/Practice2018Robot.java

```diff
@@ -27,12 +27,12 @@ public DeviceInfo getRightDriveFollower() {
     
     @Override
     public DeviceInfo getLeftDriveMasterEncoder() {
-        return new DeviceInfo(0, true);
+        return new DeviceInfo(0, false);
```

Comment:
@WasabiFan How did you determine the inversion needed to be removed? (Not that you're wrong - but I want to make sure everybody is going to use the same method so we don't whip back-and-forth on whether inversion is necessary).

Reply:
I can't recall; I switched that early in my testing. Certainly, with this change, velocity PID maintains itself rather than spinning off infinitely, so I think it's right. We can test tonight if I remember.

Reply:
Sounds good. We should verify that the PoseSubsystem isn't confused about robot motion with this inversion.